### ISSUE TITLE
ssd % not reflecting correctly

### ISSUE THREAD

**Issue by crabsinabucket** [exelban/stats#1470]
hey guys. absolutely love stats and recommend it to everybody, haven't had any issues up to now. usually ssd shows correctly the amount of space open on the hard drive, but suddenly it isn't. am i doing something incorrectly? i have already quit the process in activity monitor and restarted my machine, even did a fresh install of stats. running a 2020 m1 8gb macbook air.
<img width="271" alt="Screenshot 2023-05-16 at 16 30 15" src="https://github.com/exelban/stats/assets/130757796/ff5a7d03-ed65-469f-85a9-44f1f9ae96a1">
amount stats says is available, seems to be stuck here
<img width="470" alt="Screenshot 2023-05-16 at 16 31 41" src="https://github.com/exelban/stats/assets/130757796/dceec38f-4464-4e5a-806f-6a907f3cd7c1">
actual amount reflected in my system settings.

any idea what could be causing this?

**Comment by exelban** [comment]
stats showing the disk usage with purgeable space. Because apple broke the API that allows it to get a free space without it.

**Comment by luckman212** [comment]
Is this still broken in Sonoma? Just wondering. I came here to report this "bug" but I see, aside from this, there is also:

- https://github.com/exelban/stats/issues/1589
- https://github.com/exelban/stats/issues/1552
- https://github.com/exelban/stats/issues/1510

So I obviously won't report again.

But, here's my screenshot anyway... from 13.5.2 + Stats 2.9.7

![screenshot_mzOW43Pk](https://github.com/exelban/stats/assets/1992842/256b0228-3aab-4f75-9f5f-183c08d802ba)

**Comment by luckman212** [comment]
@exelban Is this API still broken in Sonoma? Just wondering. I came here to report this "bug" but I see, aside from this, there is also:

- https://github.com/exelban/stats/issues/1589
- https://github.com/exelban/stats/issues/1552
- https://github.com/exelban/stats/issues/1510

So I obviously won't report again.

But, here's my screenshot anyway... from 13.5.2 + Stats 2.9.7

![screenshot_mzOW43Pk](https://github.com/exelban/stats/assets/1992842/256b0228-3aab-4f75-9f5f-183c08d802ba)

**Comment by exelban** [comment]
@luckman212 it's have nothing to issues that you mention. They are related to the "incorrect" free disk space size. I assume that you have selected this disk as primary in the settings?

**Comment by luckman212** [comment]
@exelban Yes, that is my boot disk and it is selected as the main disk in settings. Should I move this to a new issue then?

Yesterday my Mac almost crashed because the disk filled up to 96% due to APFS snapshots going out of control. I think maybe because this space was technically "purgeable" maybe that's why the guage in Stats didn't report it?

I never had this bug before (nothing to do with Stats of course, just a macOS bug). I had to nuke all my local snapshots with 
```
tmutil deletelocalsnapshots /
```
Free space then went back to normal. Since then I also updated to 13.6.
